.gitignore

```diff
@@ -4,3 +4,17 @@
 ./testing/alg-theme-cala-config/alg-*
 
 ./testing/calamares-3.3.5/calamares*
+
+# Ignore from cala-configs
+./cala-configs/alg-wm-cala-config/objects/pack/*
+
+# Ignore from calamares
+./calamares/calamares-3.2.26
+./calamares/calamares-3.2.39
+./calamares/calamares-3.2.47
+./calamares/calamares-3.2.62
+./calamares/calamares-3.3.5
```

Comment:
I would add a line to ignore the extra things cmake creates when building calamares-3.3.6. something like line 6 but this folder situation

Reply:
added line:

./calamares/calamares-3.3.6/calamares*